editions/tw5tiddlyweb/tiddlywiki.info

```diff
@@ -1,7 +1,8 @@
 {
 	"description": "Base edition for TiddlySpace integration",
 	"plugins": [
-		"tiddlywiki/tiddlyweb"
+		"tiddlywiki/tiddlyweb",
+		"tiddlywiki/filesystem"
 	],
```

Comment:
hmmm. do we need them here?

Reply:
Ah, yea. the original tiddlyweb plugin needs both. .. But I don't know, what this has to do with webdav. ... 

If it's just a general fix. please consider a second PR with just this fix.